superset/db_engine_specs/presto.py

```diff
@@ -1329,7 +1329,9 @@ def handle_cursor(cls, cursor: Cursor, query: Query) -> None:
                         completed_splits,
                         total_splits,
                     )
-                    if progress > query.progress:
+                    if (  # pylint: disable=consider-using-min-builtin
```

Comment:
I opted not to use the suggestion, i.e., `query.progress = max(progress, query.progress)` as this would likely dirty the SQLAlchemy object and result in an excess of commits.